gluecodium/src/main/java/com/here/gluecodium/generator/dart/DartGeneratorSuite.kt

```diff
@@ -142,19 +137,19 @@ class DartGeneratorSuite(options: Gluecodium.Options) : GeneratorSuite() {
     }
 
     private fun collectImports(
-        rootElement: LimeNamedElement,
+        allTypes: List<LimeType>,
         importResolver: DartImportResolver
     ): List<DartImport> {
-        val functions = collectFunctions(rootElement)
-        val properties = (rootElement as? LimeContainerWithInheritance)?.properties ?: emptyList()
-        val exceptions = collectExceptions(rootElement)
-        val structs = listOfNotNull(rootElement as? LimeStruct) +
-            ((rootElement as? LimeContainer)?.structs ?: emptyList())
-        val typeRefs = functions.flatMap { collectTypeRefs(it) } + properties.map { it.typeRef } +
-            exceptions.map { it.errorType } + structs.flatMap { it.fields }.map { it.typeRef }
-
-        return importResolver.resolveImports(rootElement) +
-            functions.mapNotNull { it.exception }.flatMap { importResolver.resolveImports(it) } +
+        val functions = allTypes.filterIsInstance<LimeContainer>().flatMap { it.functions }
```

Comment:
It was a little bit more clear before change. Not it filter types by LimeContainer and it looks a little bit weird (I think since I understand not so much)